library.properties

```diff
@@ -0,0 +1,10 @@
+name=OBD9141
+version=1.0.0
+author=Ivor Wanders
+maintainer=Ivor Wanders
+sentence=Read data from the the ISO 9141-2 (K-line) pin of an OBD-II port found in cars.
+paragraph=This makes it easy to read data from the diagnostic port and display or record it.
```

Comment:
The paragraph line is apparently [prefixed](https://github.com/arduino/Arduino/wiki/Arduino-IDE-1.5:-Library-specification#libraryproperties-file-format) by the sentence, also the library itself does not display or record data, it surely helps if that is the intent of the user though, I get what you are trying to say.

Perhaps something like this describes it well:
`paragraph=This provides a class that makes it easier to interact with the port, handling the initialization and provides methods to retrieve PID values from the ECU.`